code/modules/mob/living/living.dm

```diff
@@ -787,9 +787,9 @@
 /mob/living/proc/makeTrail(turf/target_turf, turf/start, direction)
 	if(!has_gravity())
 		return
-	var/blood_exists = FALSE
+	var/blood_exists = locate(/obj/effect/decal/cleanable/blood/trail_holder) in start
 
-	for(var/obj/effect/decal/cleanable/trail_holder/C in start) //checks for blood splatter already on the floor
+	for(var/obj/effect/decal/cleanable/blood/trail_holder/C in start) //checks for blood splatter already on the floor
```

Comment:
this loop isn't necessary, since the prior line that initializes blood_exists already finds a trail_holder in start